aspnet/publishing/iis.rst

```diff
@@ -291,6 +312,17 @@ Troubleshooting
 
 - Confirm that the Application Pool is not in the `Stopped` state.
 
+IISIntegration module was not injected
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- **Browser:** 502.5 Process Failure
+- **Application Log:** Process was created with commandline '"dotnet" ...' but either crashed or did not reponse within given time or did not listen on the given port '2140', ErrorCode = '0x800705b4'
+- **ASP.NET Core Module Log:** Log file created and shows normal operation.
+
+Troubleshooting
+
+- Confirm that you have correctly referenced the IISIntegration module by calling the UseIISIntegration() method of the application's WebHostBuilder().
```

Comment:
I'm (slowly) making progress on my day's tasks. I should get to this sometime this evening. I'll test here, too, to confirm language for the Troubleshooting section and ping you back.